app/src/main/java/org/schabi/newpipe/player/ui/PlayerUiList.kt

```diff
@@ -4,7 +4,7 @@ import org.schabi.newpipe.util.GuardedByMutex
 import java.util.Optional
 
 class PlayerUiList(vararg initialPlayerUis: PlayerUi) {
-    var playerUis = GuardedByMutex(mutableListOf<PlayerUi>())
+    private var playerUis = GuardedByMutex(mutableListOf<PlayerUi>())
```

Comment:
Do this and remove the init (and move the init comment according to Kotlin's rules for constructor comments)
```suggestion
    private var playerUis = GuardedByMutex(mutableListOf<PlayerUi>(initialPlayerUis))
```

Reply:
Can this be a `val`?